allennlp/service/server_flask.py

```diff
@@ -229,6 +229,8 @@ def predict(model_name: str) -> Response:  # pylint: disable=unused-variable
                     verbs.append({"verb": verb["verb"], "description": verb["description"]})
 
             log_blob["outputs"]["verbs"] = verbs
+        elif model_name == "wikitables-parser":
+            log_blob['outputs']['logical_form'] = prediction['logical_form']
```

Comment:
Can you add the answer here, too?  That's part of the predictor output now, isn't it?

Reply:
That's added in #831.